StRoot/StPicoDstMaker/StPicoDstMaker.h

```diff
@@ -349,7 +349,7 @@ class StPicoDstMaker : public StMaker {
     return cvs;
   }
 
-  ClassDef(StPicoDstMaker, 0)
+  ClassDef(StPicoDstMaker, 1)
```

Comment:
Should stay at zero unless you plan to use streamers

Reply:
fixed in latest commit